pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -178,23 +172,18 @@ public QueryExecutor openConnectionImpl(HostSpec[] hostSpecs, String user, Strin
 
         // Set SO_RECVBUF read buffer size
         int receiveBufferSize = PGProperty.RECEIVE_BUFFER_SIZE.getInt(info);
-        if (receiveBufferSize > -1) {
```

Comment:
AFAIK `-1` was here to just use system default. Raising a warning for that case is not good

Reply:
Was that an intentional change?

Reply:
Reverted.